Make travis install puppet before running.

bltavares opened the pull request.

carlossg commented:
Hi Bruno, we are forking librarian just for fixes, you may want to submit your request to https://github.com/rodjek/librarian-puppet. You'd want to squash the commits into just one to keep history clean.

We can merge it in our maestrodev branch too anyway

bltavares commented:
Hi @carlossg , I was facing issues with librarian-puppet and noticed some mentions to your repo on the discussion.

The official librarian-puppet is having the same problem and it is not on Travis CI. I will suggest this with a squashed pull commit. I can close this pull request then.

Thank you for the informantion (:

bltavares committed:
Introduces puppet as a testing dependency to pass on Travis Ci
Squashed commit of the following:

commit 27f67b77823e3fe4bf3bbffc7eae47bbc12ea88f
Author: Bruno Tavares <bruno.exz@gmail.com>
Date:   Wed Mar 6 07:28:39 2013 -0300

    Include build status on the README

commit 45dfc7bf2178a99debc26b40995ec217111ee599
Author: Bruno Tavares <bruno.exz@gmail.com>
Date:   Tue Mar 5 21:33:47 2013 -0300

    Increase timeout on the tests. The internet is not always fast everywhere

commit 578d52f2612a72d48aeca709f2a6a99cbff899af
Author: Bruno Tavares <bruno.exz@gmail.com>
Date:   Tue Mar 5 21:26:42 2013 -0300

    Set the gem file to include puppet on the tests, so bundler will include in the bundle exec

commit ab739f5429be273fdc4819942fd9c03195894ea4
Author: Bruno Lara Tavares <bruno.exz@gmail.com>
Date:   Tue Mar 5 21:14:12 2013 -0300

    Make travis install puppet before running.

bltavares commented:
Hi @carlossg , just to keep you in the loop, I am sending this pull request to rodjek/librarian-puppet#80

Have a nice day (:
